daily-shoutout/app.py

```diff
@@ -55,11 +56,22 @@ def sendTweet(tweet_string):
     client.create_tweet(text = tweet_string)
 
 
+def returnResponse(response_string):
+    return{
+        "statusCode": 200,
+        "body": json.dumps(response_string)
+    }
```

Comment:
Do you want to return a `json` dump of this whole response? Something like this?

```suggestion
    return json.dumps({
        "status_code": 200,
        "body": response_string,
    })
```

Reply:
So in the context here it will be the response to an api endpoint.  So we would want to return a statusCode: 200 OK, but also the response body as json.  If all of it is in json, I don't think that the browser will be able to see that the call was successful.  i can test because now I'm curious.